### PR TITLE
feat(core): add support for multi positional args

### PR DESCRIPTION
**Pull request by jerome-quere**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request.
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

### CODE REVIEW

**Comment by jerome-quere** [comment]
Need to find how to handle json response when there are multiple positional args

File: internal/core/cobra_utils.go [hunk 1]
```diff
@@ -27,132 +27,133 @@ func cobraRun(ctx context.Context, cmd *Command) func(*cobra.Command, []string)
...
 			}
-			waitFlag, err := cobraCmd.PersistentFlags().GetBool("wait")
-			if err == nil && cmd.WaitFunc != nil && waitFlag {
-				data, err = cmd.WaitFunc(ctx, cmdArgs, data)
-				if err != nil {
-					return err
-				}
+
+			results = append(results, result)
+			err = meta.Printer.Print(result, nil)
```
Comment (Sh4d1) [inline internal/core/cobra_utils.go:83]: `opt := cmd.getHumanMarshalerOpt()` is missing here

File: internal/core/result.go [hunk 2]
```diff
@@ -67,3 +69,17 @@ func (s *SuccessResult) getMessage() string {
...
+func (mr MultiResults) MarshalHuman() (string, error) {
+	strs := []string(nil)
+	for _, r := range mr {
+		str, err := human.Marshal(r, nil)
+		if err != nil {
+			return "", err
+		}
+		strs = append(strs, str)
+	}
+	return strings.Join(strs, "\n"), nil
```
Comment (Sh4d1) [inline internal/core/result.go:84]: Should we maybe add two new lines here?

Comment (Sh4d1) [reply internal/core/result.go:84]: Or another kind of delimiter :thinking: Right now it looks like:
```
Volumes:
ID                                    NAME                                                            EXPORT URI        SIZE    VOLUME TYPE  CREATION DATE  MODIFICATION DATE
9faa0cfe-3119-430c-83d5-c6241472526f  snapshot-e845ae67-2e67-4ec0-8955-3293a851b092-2018-08-31_12:09  device://dev/vda  100 GB  l_ssd        1 year ago     1 year ago
Server:
```

**Pull request merged**